### PR TITLE
Adding some new flavors tests

### PR DESCRIPTION
**Pull request by rnirmal**
Added some missing flavors tests.

### CODE REVIEW

File: tests/integration/tests/api/flavors.py [hunk 1]
```diff
@@ -18,16 +18,18 @@
...
 from proboscis import test
+from proboscis.asserts import assert_raises
 
 import tests
 from tests.util import create_dbaas_client
 from tests.util import create_nova_client
 from tests.util import test_config
 from tests.util.users import Requirements
-
+from tests.util.check import AttrCheck
```
Comment (cp16net) [inline tests/integration/tests/api/flavors.py:32]: oh neato

**Comment by cp16net** [comment]
+1

**Comment by ed-** [comment]
+1

**Comment by TimSimpsonR** [comment]
+1

**Comment by hub-cap** [comment]
+1

**Pull request merged**